gcc/rust/rust-session-manager.cc

```diff
@@ -477,66 +473,41 @@ Session::parse_file (const char *filename)
       break;
     }
 
-  /* basic pipeline:
-   *  - lex
-   *  - parse
-   *  - register plugins (dummy stage for now) - attribute injection? what is
-   * this? (attribute injection is injecting attributes specified in command
-   * line into crate root)
-   *  - injection (some lint checks or dummy, register builtin macros, crate
-   * injection)
-   *  - expansion (expands all macros, maybe build test harness, AST validation,
-   * maybe macro crate)
-   *  - resolution (name resolution, type resolution, maybe feature checking,
-   * maybe buffered lints)
-   *  TODO not done */
-
-  fprintf (stderr, "\033[0;31mSUCCESSFULLY PARSED CRATE \n\033[0m");
-
   // register plugins pipeline stage
   register_plugins (parsed_crate);
-  fprintf (stderr, "\033[0;31mSUCCESSFULLY REGISTERED PLUGINS \n\033[0m");
-
   if (options.dump_option == CompileOptions::REGISTER_PLUGINS_DUMP)
     {
-      // TODO: what do I dump here?
+      // TODO:
     }
 
   // injection pipeline stage
   injection (parsed_crate);
-  fprintf (stderr, "\033[0;31mSUCCESSFULLY FINISHED INJECTION \n\033[0m");
-
   if (options.dump_option == CompileOptions::INJECTION_DUMP)
     {
-      // TODO: what do I dump here? injected crate names?
+      // TODO:
     }
 
   // expansion pipeline stage
   expansion (parsed_crate);
-  fprintf (stderr, "\033[0;31mSUCCESSFULLY FINISHED EXPANSION \n\033[0m");
-
   if (options.dump_option == CompileOptions::EXPANSION_DUMP)
     {
-      // dump AST with expanded stuff
-      fprintf (stderr, "BEGIN POST-EXPANSION AST DUMP\n");
-      parser.debug_dump_ast_output (parsed_crate);
```

Comment:
I think that this expansion dump should be kept, at least temporarily if you are changing the way dumps work. It is useful to compare the results of the AST pre-macro expansion and post-macro expansion.

Reply:
Sounds good i will keep that in. I want to make a change separate to this to allow multiple dump options to be specified but it will be a separate PR

Reply:
I will revert these changes actually for now and add in the scan or unused to make this change cleaner.

Reply:
Reverted thanks again!